core/source/org/libreoffice/ide/eclipse/core/wizards/pages/.gitignore

```diff
@@ -0,0 +1 @@
+/UnoPackageExportPage.java
```

Comment:
Why this?

Reply:
Sorry, this file got created and pushed, would remove it